torch/testing/_internal/custom_op_db.py

```diff
@@ -83,9 +86,116 @@ def numpy_take_impl(x, ind, ind_inv, dim):
     ind = to_numpy(ind)
     return torch.tensor(np.take_along_axis(x, ind, dim), device=device)
 
-@numpy_take.impl_meta()
-def numpy_take_meta(x, ind, ind_inv, dim):
-    return x
+@numpy_take.impl_fake()
+def numpy_take_fake(x, ind, ind_inv, dim):
+    return torch.empty_like(x)
+
+@custom_op('(Tensor x) -> Tensor', ns='_torch_testing')
```

Comment:
I'm getting some buyer's remorse from removing the deduplication; the main downside is I can no longer grep _torch_testing::numpy_nonzero to find the operator that defines this lol

Reply:
Well, we can bring the name back and just assert that the name mentioned in the schema matches the function name:

```
# with schema
@custom_op('_torch_testing::numpy_nonzero', '(Tensor x) -> Tensor')
def numpy_nonzero(x):
    ...

# without schema (inferring schema from type annotations)
@custom_op('_torch_testing::numpy_nonzero')
def numpy_nonzero(x: Tensor) -> Tensor:
    ...
```

Reply:
(will leave for the future)